docs/server_notices.md

```diff
@@ -61,14 +61,33 @@ displayname and avatar of the Server Notices user.
 Sending notices
 ---------------
 
-As of the current version of synapse, there is no convenient interface for
-sending notices (other than the automated ones sent as part of consent
-tracking).
+The API to send notices is as follows:
 
-In the meantime, it is possible to test this feature using the manhole. Having
-gone into the manhole as described in [manhole.md](manhole.md), a notice can be
-sent with something like:
+```
+POST /_matrix/client/r0/admin/send_server_notice/[<user_id>]
+```
+
+including an `access_token` of a server admin.
 
+If the user_id is missing, the message is meant to go to all users on the server
```

Comment:
I feel like sending a message to all users is a dangerous thing that might happen by accident. Would it be better to have separate APIs

`/admin/send_server_notice_to_user/<user_id>`

`/admin/send_server_notice_to_all_users`

?

Reply:
well, fair enough. I won't insist.